tokio/src/sync/mpsc/unbounded.rs

```diff
@@ -172,6 +172,41 @@ impl<T> UnboundedReceiver<T> {
         poll_fn(|cx| self.poll_recv(cx)).await
     }
 
+    /// Receives the next values for this receiver and extends `buffer`.
+    ///
+    /// This method returns the number of values populated in `buffer`;
+    /// it returns 0 if the channel has been closed and there
+    /// are no remaining messages in the channel.
+    /// If there are no messages in the channel's buffer, but the channel has
+    /// not yet been closed, this method will sleep until a message is sent or
+    /// the channel is closed.
+    ///
+    /// If at the time of the call the buffer has no unused capacity,
+    /// `BLOCK_CAP` additional elements are reserved.
```

Comment:
The user doesn't know what `BLOCK_CAP` is because it's not a public constant. I wouldn't explain how much space we reserve. That will let us change it in the future.

Reply:
fixed in latest push.